test/integration/scripting_spec.js

```diff
@@ -191,4 +191,79 @@ describe("Interaction", () => {
       );
     });
   });
+
+  describe("in js-buttons.pdf", () => {
+    let pages;
+
+    beforeAll(async () => {
+      pages = await loadAndWait("js-buttons.pdf", "#\\38 0R");
+    });
+
+    afterAll(async () => {
+      await closePages(pages);
+    });
+
+    it("must show values in a text input when clicking on radio buttons", async () => {
+      await Promise.all(
+        pages.map(async ([browserName, page]) => {
+          const expected = [
+            ["#\\36 8R", "Group1=Choice1::1"],
+            ["#\\36 9R", "Group1=Choice2::2"],
+            ["#\\37 0R", "Group1=Choice3::3"],
+            ["#\\37 1R", "Group1=Choice4::4"],
+          ];
+          await page.waitForTimeout(100);
```

Comment:
Same here, why is this timeout needed is there an event that can be used instead?